src/js/video.js

```diff
@@ -546,6 +548,20 @@ videojs.xhr = xhr;
 videojs.TextTrack = TextTrack;
 
 /**
+ * AudioTrack class
```

Comment:
Could you add comments about why these two classes are available publicly? (I assume it's so source handlers can create and add them at runtime)